packages/firestore/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "prebuild": "tsc -d --downlevelIteration --declarationDir dist/lib --emitDeclarationOnly && tsc -m es2015 --moduleResolution node scripts/*.ts ",
     "build": "rollup -c",
+    "build:deps": "lerna run --scope @firebase/'{app,firestore}' --include-dependencies build",
```

Comment:
I'm surprised that `app` isn't picked up by `--include-dependencies`. Does it not get picked up because it's a peer dependency?

Reply:
Yup. Will have to put it in the devDepedencies for lerna to pick it up, but it has an undesired side effect that it triggers Firestore version bump when `app` changes.